packages/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -262,117 +261,6 @@ describe('validation logic', () => {
       );
     });
 
-    describe('from', () => {
```

Comment:
`FROM`, `STATS`, and `METRICS` command tests move to their separate respective files and test cases grouped using `describe()` statements. Otherwise, this file contained 5K test cases, which takes while to run and impractical terminal output.